mcs/class/monodoc/Monodoc.Ecma/EcmaDesc.cs

```diff
@@ -263,9 +255,18 @@ void ConstructCRef (StringBuilder sb)
 			if (DescKind == Kind.Type)
 				return;
 
+			if (ExplicitImplMember != null) {
+				sb.Append ('$');
```

Comment:
> do we have ToEcmaCref conform to the already-implemented functionality in monodoc, or do we change the implementation of the `EcmaHelpSource.GetPublicUrl` to comform to the points you've brought up?

I'm inclined to say that we should fix `EcmaHelpSource.GetPublicUrl()`, except for one minor problem: `#` is the anchor for URLs, so a URL such as this has little chance of making sense:

    http://docs.go-mono.com/?link=M:System.Collections.Generic.Dictionary<TKey,TValue>.System#Collections#Generic#ICollection<System.Collections#Generic.KeyValuePair<TKey,TValue>>#Add

To even begin to make that work, you'd have to change the URL scheme to be...something else. (Wasn't there some "newfangled" URL convention a few years ago where everything of interest was after the anchor?)

So I'm torn: for XML/file use, `#` should be used, but for URLs... I'm not sure.

Perhaps replace `#` with `-`?